run_array_analyzer.py

```diff
@@ -25,7 +26,14 @@ def parse_args():
         help="Output directory path",
     )
     parser.add_argument(
-        '-m', '--method',
+        '-f', '--format',
+        type=str,
+        choices=['well', 'array'],
+        default='array',
```

Comment:
Creating a workflow per assay format (and corresponding data format) is a good idea. It allows reuse of modules, e.g. compute_OD can be reused in this case. It also makes maintenance of each format easier. 

Instead of `--format well` and `--format array`, I suggest more obvious `--fomat elisa` and `--format scienion-array` flags.

Reply:
I would vote for "elisa" and "array" So that we do not have hyphens in the argument name itself.

Reply:
The same workflow can also be used for non-ELISA measurements, so I thought that something like `well` and `array` are fitting. Thought the workflows may require some modifications between different platforms - in that case platform specific naming (like `96wp_hummingbird`, `scienion_array`, or 'octopi_array') would be more appropriate.

Reply:
`--format` flag  connotes antigen array format. `elisa` and `array` are good values. If we extend analysis to echo or luminex formats, we can change these values.

Single workflow should be able to handle data from different readers.  The parameters of the imagers (hummingbird, Scienion printer, octopi) are captured by the configuration metadata.

Reply:
How about this? https://github.com/czbiohub/serology-COVID19/blob/aff4081e1e94d5d89de6c970230a739c03d3461d/run_array_analyzer.py#L64-L90